### PR TITLE
README: install instructions for Solidity and DappHub Toolkit

### LINKED ISSUE #226: Audit the documentation generated in #224
Make sure you can (record anywhere that there is confusion):

- [x] Install KEVM on a fresh container following instructions.
- [x] Build KLab and run the SafeAdd example following the instructions.
- [ ] Build the k-dss repo and run the proofs there following the instructions.
- [x] Build the mkr-mcd-spec repo and run the tests there following the instructions.

### PR DESCRIPTION
**Pull request by ehildenb**
Fixes: https://github.com/makerdao/mkr-mcd-spec/issues/226

This adds explicit instructions for setting up the DappHub toolkit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,26 @@ If you want to build KEVM and KLab from source, you can:
...
+```sh
+nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions
+```
+
+And finally, install the [DappHub toolkit](https://dapp.tools/).
+
+```sh
+curl https://dapp.tools/install | sh
+```
+
```
Comment (anvacaru) [inline README.md]: These should go at the top of `Instalation & Running`.

**Comment by anvacaru** [comment]
There is still a typo at line 25 in README.md: `$(pwd)/deps/evm-semanticss`.

File: README.md (hunk 1 above)
Comment (ehildenb) [reply README.md]: Good idea, fixed.

**Comment by ehildenb** [comment]
All updates to documentation applied. This should be ready for merge.

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**